src/mcts/node.h

```diff
@@ -378,25 +378,42 @@ class EdgeAndNode {
 
   // Edge related getters.
   float GetP() const { return edge_->GetP(); }
+  float GetPEffect(float policy_factor, float policy_factor_parent,
+                  float policy_exponent) const {
+    auto visits = GetN();
+    auto visits_parent = node_ ? node_->GetParent()->GetN() : 0;
+    auto psa = GetP();
+    return ( ( policy_factor > 0.0 ) || ( policy_factor_parent > 0.0 ) )
+      ? ( psa > 0.0f
+        ? 1.0f / ( 1.0f + (1.0f / psa - 1.0f) *
+            FastPow( 1.0f + policy_factor * visits +
+                          policy_factor_parent * visits_parent, policy_exponent )
```

Comment:
Unless I'm missing something (not unlikely), this should be `/ FastPow()` instead of `* FastPow()` to be equivalent to #1173. And no, `FastPow()` **will not** work with negative exponents.

Reply:
You are completely correct there :)

Reply:
Yep. I just realized this. Thanks guys.

Reply:
I've updated the benchmark above.